includes/class-customizer.php

```diff
@@ -316,20 +297,73 @@ public static function customize_save_customizer( $wp_customize ) {
 					$featured_items = json_decode( $theme_mod );
 					$old_to_new_id  = array();
 
+					/**
+					 * Loop through all currently published featured items in this area
+					 * and add them to an array keyed by original post id.
+					 * This is so that we can update that post below
+					 * instead of trashing all posts and then creating new ones.
+					 */
+					$query = new \WP_Query(
+						array(
+							'post_type'      => 'featured-content',
+							'post_status'    => 'publish',
```

Comment:
Now, when we do introduce future as a `post_status` by design for featured content this argument should contain an array with both `publish` and `future` as values.

```
'post_status'    => [
	'publish',
	'future',
],
```

Reply:
fixed in dfbd5b1